### PR TITLE
Add additional Maven options

### PR DESCRIPTION
**Pull request by celanthe**
Added additional Maven options as was suggested in #9

### CODE REVIEW

File: resources/prepare-next-development-version.sh [hunk 1]
```diff
@@ -21,8 +21,11 @@ git commit -am "release(v${RELEASE_VERSION})"
 git tag -fa "${RELEASE_VERSION}" -m "release(v${RELEASE_VERSION})"
 git push origin --tags -f
 
-mvn -B org.apache.maven.plugins:maven-release-plugin:update-versions -DgenerateBackupPoms=false
-
+  mvn -B ${{ maven-additional-options:
```
Comment (gclaussn) [inline resources/prepare-next-development-version.sh:24]: Same here, the inlining will not work, the additional options must be provided as thrid argument when the script is called:
https://github.com/camunda-community-hub/community-action-maven-release/blob/d0e538b36d39612c03b85cff3475063933a0d5f0/action.yml#L93

The new script call:
```
run: ${{ github.action_path }}/resources/prepare-next-development-version.sh "${{ github.event.repository.default_branch }}" "${{ inputs.release-version }}" "${{ inputs.maven-additional-options }"
```

In the script, check for the third argument, if it exists:
```
MAVEN_ADDITIONAL_OPTIONS=$3
```

The mvn call should look like this:
```
mvn -B ${MAVEN_ADDITIONAL_OPTIONS} org.apache.maven.plugins:maven-release-plugin:update-versions -DgenerateBackupPoms=false
```

Comment (lwille) [reply resources/prepare-next-development-version.sh:24]: thanks @gclaussn, you explained it perfectly!

Comment (celanthe) [reply resources/prepare-next-development-version.sh]: Thank you so much for the feedback @gclaussn! I made two more PRs that I hope put things in the right place. The only other thing I'm struggling with is where to put the new mvn call. Is there a specific line that `mvn -B ${MAVEN_ADDITIONAL_OPTIONS} org.apache.maven.plugins:maven-release-plugin:update-versions -DgenerateBackupPoms=false` should go on in `action.yml`? or does it go in the .sh code? 

Thank you again for any help you can provide. :) I very much appreciate it!

**Comment by celanthe** [comment]
closing in favor of #13

**Pull request closed without merging**